File: Java/기본 수학2/소수.java
import java.io.*;

public class Main{
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int M = Integer.parseInt(br.readLine());
		int N = Integer.parseInt(br.readLine());
		int sum = 0; int cnt = 0; int min = 0;

		for(int i = M; i < N+1; i++) {
			int tmp = 0;
			
			if(i > 1) {
				for(int j = 2; j < i; j++) {
					if(i % j == 0) {
						tmp = -1;
						break;
					}
				}
				if(tmp == 0) {
					sum += i;
					if(cnt == 0) min = i;
					cnt++;
				}
			}
		}
		
		if(min == 0) System.out.print(-1);
		else {
			System.out.println(sum);
			System.out.println(min);
		}
    }
}

File: C++/기본 수학2/소수.cc
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int a, b, check = 0, min = 0, sum = 0;
    cin >> a >> b;
    
    for(int i = a; i <= b; i++) {
        for(int j = 1; j < i; j++) {
            if(i % j == 0)
                check++;
        }
        if(check == 1) {
            if(min == 0)
                min = i;
            sum += i;
        }
		check = 0;
    }
    
    if(sum == 0)
        cout << -1;
    else
        cout << sum << '\n' << min;
   
    return 0;
}